charts/rustrial-aws-eks-iam-auth-controller/values.yaml

```diff
@@ -67,3 +67,17 @@ tolerations: []
 affinity: {}
 
 topologySpreadConstraints: {}
+
+defaultIdentity:
```

Comment:
@levtomer66 IMHO, we should not add the `defaultIdentity` part and only focus on the `additionalIdentities` part. The `defaultIdentity` is too specific and the same functionality can be achieved using `additionalIdentities`, for example like this: 

```yaml
additionalIdentities:
  - name: "aws-ec2-nodes"
    arn: "arn:aws:iam::{{AccountID}}:role/my-eks-node-instance-profile"
    username: "system:node:{{EC2PrivateDNSName}}"
    groups:
    - system:bootstrappers
    - system:nodes
```

If we want to make this more obvious to end-users we can kind of document it like this in `values.yaml`:

```yaml
additionalIdentities: []
#  - name: "aws-ec2-nodes"
#    arn: "arn:aws:iam::{{AccountID}}:role/my-eks-node-instance-profile"
#    username: "system:node:{{EC2PrivateDNSName}}"
#    groups:
#    - system:bootstrappers
#    - system:nodes
```

Reply:
Sure, fixed 👍🏼